src/blog/2025/08/annual_billing.md

```diff
@@ -21,7 +21,8 @@ Switching to annual billing is straightforward:
 
 **New customers:** Select annual billing when signing up for Starter or Pro plans
 
-**Existing customers:** Switch to annual billing from your Billing settings
+**Existing customers:** Switch to annual billing from your Billing settings. Navigate to Billing, then choose Upgrade Team, select your team type, and a toggle will appear to chose Yearly billing.

```

Comment:
Your existing team plan is already highlighted/selected, they just need to switch the yearly toggle to enabled